post_installation_configuration/cluster-tasks.adoc

```diff
@@ -580,3 +580,22 @@ Understand and configure pod disruption budgets.
 
 include::modules/nodes-pods-pod-disruption-about.adoc[leveloffset=+2]
 include::modules/nodes-pods-pod-disruption-configuring.adoc[leveloffset=+2]
+
```

Comment:
These changes have been reviewed in the other PR for 4.7 CCO work, but I copied them over since that one isn't merged yet.